pkg/json_raw_results.go

```diff
@@ -63,7 +63,7 @@ type jsonBranchProtectionSettings struct {
 	RequiresLinearHistory               *bool    `json:"requiredLinearHistory"`
 	DismissesStaleReviews               *bool    `json:"dismissesStaleReviews"`
 	EnforcesAdmins                      *bool    `json:"enforcesAdmin"`
-	RequiresStatusChecks                *bool    `json:"requiresStatuChecks"`
+	RequiresStatusChecks                *bool    `json:"requiresStatusChecks"`
```

Comment:
⚠️  I'm worried that this is a breaking change. I'm happy to drop this if that's the case.

Reply:
didn't dig into it too much, but let's just drop it